.github/workflows/pipeline.develop.yml

```diff
@@ -137,6 +137,29 @@ jobs:
           API_KEY: ${{ secrets.API_KEY_DEVELOP }}
           CHAIN_ID: ${{ secrets.CHAIN_ID_AVAX }}
         run: npx hardhat run scripts/proof-verification.ts
+      - name: Unit tests for Arbitrum
+        env:
+          BLOCKCHAIN_PROVIDER_URL: ${{ secrets.BLOCKCHAIN_PROVIDER_URL_ARBITRUM_DEVELOP }}
```

Comment:
Can't we manipulate `secrets` per environment? That way we don't need to use `_DEVELOP` on every env.